lib/view/item_widget.dart

```diff
@@ -19,11 +20,28 @@ class _ItemWidgetState extends State<ItemWidget> {
   Widget build(BuildContext context) {
     final itemTextStyle = Theme.of(context).textTheme.titleLarge;
     return ListTile(
+      onLongPress: _showConfirmDeleteModalSheet,
       title: Text(widget.item.value, style: itemTextStyle),
       onTap: _showEditDialog,
     );
   }
 
+  void _showConfirmDeleteModalSheet() {
+    showModalBottomSheet(
+      context: context,
+      builder: (context) => Row(
+        children: [
+          TextButton(
+              onPressed: () {
+                widget.onDelete();
+                Navigator.pop(context);
+              },
+              child: const Icon(Icons.delete)),
+        ],
+      ),
```

Comment:
```suggestion
    builder: (context) => Padding(
        padding: const EdgeInsets.all(8.0),
        child: Row(
          mainAxisAlignment: MainAxisAlignment.spaceBetween,
          children: [
            TextButton.icon(
                onPressed: () {
                  widget.onDelete();
                  Navigator.pop(context);
                },
                icon: const Icon(Icons.delete),
                label: const Text('Delete'),
                style: const ButtonStyle(
                    foregroundColor:
                        MaterialStatePropertyAll<Color>(Colors.red)))
          ],
        ),
      ),
```

Reply:
Add some padding around your content, so it doesn't look so squished. Use an icon with text, colored red to show that it is a "danger" action.

Reply:
That makes it look a lot better.

I am concerned, though, that if we add more buttons with labels, the modal sheet's row might overflow. Do you want me to look into adding some conditional visibility to the label?

Reply:
I wouldn't worry about it for now. Rows can wrap, they don't have to overflow. If we have to add more buttons, we can look at ways to show them all well then.